pkg/eestream/encryption.go

```diff
@@ -48,7 +48,8 @@ func (nonce *Nonce) Increment(amount int64) (truncated bool, err error) {
 // AESGCMNonce returns the nonce as a AES-GCM nonce
 func (nonce *Nonce) AESGCMNonce() *AESGCMNonce {
 	aes := new(AESGCMNonce)
-	copy((*aes)[:], (*nonce)[:AESGCMNonceSize])
+	// Lower bytes should be taken instead of higher bytes, so nonce.Increment has effect
+	copy((*aes)[:], nonce[AESGCMNonceSize:])
```

Comment:
Is there a reason we are not doing the increment from the other side?

Reply:
You right. It can be done clearer. Changed the starting position to `NonceSize - AESGCMNonceSize`.

The reason we want the lower bytes is due to how the `eestream.incrementBytes` function is implemented.

Reply:
Is there a reason incrementBytes needs to be big-endian?